<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultShowWindow</class>
 <widget class="QDialog" name="ResultShowWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>505</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Опорный план</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView_basic_plan"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Результат оптимизации</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView_res"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Итоговая стоимость перевозки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="result_price_line_edit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QTextBrowser" name="view_temperaly_calculation">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Ход решения</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
